### PR TITLE
Removed API endpoints and other code related to pricing/payment plans…

### PR DESCRIPTION
**Pull request by vraravam**
… - since Ferdium is completely FOSS

Upgraded to the latest version of the recipes submodule

### CODE REVIEW

File: app/Controllers/Http/StaticController.js [hunk 1]
```diff
@@ -12,61 +12,19 @@ class StaticController {
...
       isMagicBarEnabled: true,
-      isTeamManagementIncludedInCurrentPlan: true,
       isTodosEnabled: true,
-      isTodosIncludedInCurrentPlan: true,
-      defaultTrialPlan: 'franz-pro-yearly',
       subscribeURL: 'https://ferdium.org',
-      planSelectionURL: 'https://ferdium.org',
       hasInlineCheckout: true,
       isPlanSelectionEnabled: false,
       isTrialStatusBarEnabled: false,
```
Comment (balajiv113) [inline app/Controllers/Http/StaticController.js:26]: Even this is not needed right ?? 

Above isPlanSelectionEnabled as well

Comment (vraravam) [reply app/Controllers/Http/StaticController.js:26]: yes - will remove both as well

File: app/Controllers/Http/StaticController.js [hunk 2]
```diff
@@ -12,61 +12,17 @@ class StaticController {
...
       serviceLimitCount: 0,
-      isCommunityRecipesPremiumFeature: false,
-      isCommunityRecipesIncludedInCurrentPlan: true,
-      isCustomUrlIncludedInCurrentPlan: true,
       isMagicBarEnabled: true,
-      isTeamManagementIncludedInCurrentPlan: true,
       isTodosEnabled: true,
-      isTodosIncludedInCurrentPlan: true,
-      defaultTrialPlan: 'franz-pro-yearly',
-      subscribeURL: 'https://ferdium.org',
```
Comment (SpecialAro) [inline app/Controllers/Http/StaticController.js:34]: This line also exists in https://github.com/ferdium/ferdium-app/blob/8ab1d10fcc1a120c031de7180019271c1c012335/src/internal-server/app/Controllers/Http/StaticController.js#L15

we should probably also delete it in the main repo

**Review by SpecialAro — APPROVED** [review]
LGTM!

File: app/Controllers/Http/StaticController.js (hunk 2 above)
Comment (vraravam) [reply app/Controllers/Http/StaticController.js:34]: Done - thanks for pointing this out. Both the code blocks are now reconciled.

**Pull request merged**